src/styles/variables.js

```diff
@@ -9,6 +9,7 @@ export default {
     fontSizeNormal: 15,
     fontSizeLarge: 17,
     fontSizeh1: 19,
+    iconSizeSmall: 20,
```

Comment:
Why `Small`? This might actually end up being our default icon size as I do think that most icons we've used so far have been at 20 x 20. So maybe it's just `iconSizeNormal`